clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -1471,8 +1471,12 @@ Error extractBundledObjects(StringRef Filename, const ArgList &Args,
           llvm::MemoryBuffer::getFileOrSTDIN(*UnbundledFile, /*isText=*/true);
       if (std::error_code EC = ObjList.getError())
         return createFileError(*UnbundledFile, EC);
-      (*ObjList)->getBuffer().split(ObjectFilePaths, '\n', /*MaxSplit=*/-1,
-                                    /*KeepEmpty=*/false);
+      // Create a copy of the list we can reference even after we close
+      // the file.
+      StringRef UnbundledArchiveList =
+          Args.MakeArgString((*ObjList)->getBuffer());
+      UnbundledArchiveList.split(ObjectFilePaths, '\n', /*MaxSplit=*/-1,
```

Comment:
Where will this variable be used? Not a blocking question.

Reply:
Thanks for the review. It's used [here](https://github.com/intel/llvm/pull/15472/files#diff-f7dc51acede945c7c8481147e773e9c09230a93b937bee9eab08ee52826cfa8eR1483), we immediately split it into the `ObjectFilePaths` vector .

Reply:
Ah..of course..I was naive and was looking for this variable on the RHS of an assignment statement. :-(